Testing/ORKTest/ORKTest/ORKESerialization.m

```diff
@@ -866,6 +867,13 @@ @implementation ORKESerializer
         },
         (@{
           })),
+  ENTRY(ORKLocationAnswerFormat,
+        ^id(NSDictionary *dict, ORKESerializationPropertyGetter getter) {
+            return [[ORKLocationAnswerFormat alloc] init];
+        },
+        (@{
+          PROPERTY(useCurrentLocation, NSNumber, NSObject, NO, nil, nil)
```

Comment:
Change writeAfterInit from `NO` to `YES`, it will fix the unit test failure in ORKTest